### PR TITLE
docs(odin-html): emmet.md Turkish translation added 

### PR DESCRIPTION
**Pull request by Doguhannilt**
# Description

Translation of emmet.md into the Turkish language

### Checklist

- [ x] discord username: `doguhan_i`
- [ ] Closes #<issue-number>
- [ ] PR must be created for an issue from issues under "In progress" column from [our project board](https://github.com/orgs/kamp-us/projects/2/views/1).
- [ x] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules, and should include the app/package/service name in the title. For example, a title like "docs(@kampus-apps/pano): Add README.md" can be used.
- [x ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [ ] I ran `npx turbo run` at the root of the repository, and build was successful.
- [ ] I installed the npm packages using `npm install --save-exact <package>` so my package is pinned to a specific npm version. Leave empty if no package was installed. Leave empty if no package was installed with this PR.

### How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

### CODE REVIEW

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 1]
```diff
@@ -1,64 +1,62 @@
...
 
-### Assignment
+### Ödev
 
 <div class="lesson-content__panel" markdown="1">
 
-1.  [This](https://www.youtube.com/watch?v=V8vizNQKtx0) is a good video on Emmet. Go ahead and watch it up to `11:37` as it demonstrates some content we haven't covered yet. We'll get there eventually.
-2.  Take a look at the [Emmet cheat sheet](https://docs.emmet.io/cheat-sheet/). You don't have to memorize it, but it's good that you get familiar with the different ways it can be used.
-3.  Go through the [Emmet documentation](https://docs.emmet.io/). Once again, no need to memorize everything but it would be good if you played around with all the different possibilities.
+1.  [This](https://www.youtube.com/watch?v=V8vizNQKtx0) , Emmet için iyi bir videodur. Devam edin ve `11:37`'e kadar izleyiniz çünkü henüz kapsamadığımız bazı içerikleri gösteriyor. Zamanla o konuya da geleceğiz.
```
Comment (cansirin) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md]:
```suggestion
1.  [Bu video](https://www.youtube.com/watch?v=V8vizNQKtx0), Emmet için iyi bir videodur. Devam edin ve `11:37`'e kadar izleyiniz çünkü henüz kapsamadığımız bazı içerikleri gösteriyor. Zamanla o konuya da geleceğiz.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 2]
```diff
@@ -1,64 +1,62 @@
...
+2.  [Emmet cheat sheet](https://docs.emmet.io/cheat-sheet/) 'ye bir göz atın. Onu ezberlemeniz gerekmiyor, ancak farklı kullanım şekilleriyle tanışmanız iyi olacaktır.
```
Comment (cansirin) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md]:
```suggestion
2.  [Emmet cheat sheet](https://docs.emmet.io/cheat-sheet/)'e bir göz atın. Onu ezberlemeniz gerekmiyor, ancak farklı kullanım şekilleriyle tanışmanız iyi olacaktır.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 3]
```diff
@@ -1,64 +1,62 @@
...
+3.  [Emmet documentation](https://docs.emmet.io/). 'ni inceleyin. Yine, her şeyi ezberlemeniz gerekmiyor, ancak farklı olanaklarla oynamak iyi olacaktır.
```
Comment (cansirin) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md]:
```suggestion
3.  [Emmet documentation](https://docs.emmet.io/)'ı inceleyin. Yine, her şeyi ezberlemeniz gerekmiyor, ancak farklı olanaklarla oynamak iyi olacaktır.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 4]
```diff
@@ -1,48 +1,46 @@
-### Introduction
+### Giriş
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md:5]:
```suggestion
---
title: 'Tablolar'
---

### Giriş
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 5]
```diff
@@ -1,48 +1,46 @@
...
+1. MDN'den [Tables Basics](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics) ve [Tables Advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced) öğreticilerini okuyunuz. Bu makaleler HTML tablolarının arkasında olan tüm sözdizimini öğretecektir ve fazlasıyla kolaydır. Okuduğunuz makalelerle birlikte kodladığınızdan emin olun!
+2. MDN'den [Tables Assessment](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Structuring_planet_data) yapınız. Yeni öğrendiğiniz becerileri pratiğe dökmek güzel!
 
 </div>
 
-### Knowledge check
+### Bilgi ölçme
 
-This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+Bu bölümde, bu dersi anladığınızı kendi başınıza kontrol etmeniz için sorular bulunmaktadır. Eğer ki bir soruyu cevaplandırmakta bir sıkıntı çekerseniz, tıklayın ve bağlantı verdiği materyali inceleyin.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md]:
```suggestion
Bu bölüm, bu dersi kendi kendinize anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve bağlantılı olduğu materyali gözden geçirin. Bu bölümde bulunan materyallerin hepsi ingilizcedir.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 6]
```diff
@@ -1,48 +1,46 @@
...
+Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
 
--   Pencil & Paper published a great [article about things you can achieve with tables](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) if you dare to spend some time giving them some love. They suggest some good habits on how you should format your data which is pretty simple and makes a huge difference. These are not a list of must dos to a table, but ideas worth keeping in mind for the next time a table is bothering you with how it looks. 
+- Eğer onlara biraz sevgi vererek biraz zaman ayırmaya cesaretin varsa, Pencil & Paper harika bir [article about things you can achieve with tables](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) yayınladı.  Verilerinizi nasıl biçimlendirmeniz gerektiği konusunda basit ancak büyük farklar yaratabilecek iyi alışkanlıklar öneriyorlar. Bu, bir tabloya karşı bir sonraki sefer sizi rahatsız ediyorsa akılda bulundurulmaya değer fikirlerdir; bunlar bir tablo için mutlaka yapılması gerekenlerin bir listesi değil, ancak düşünmeye değer fikirlerdir.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md]:
```suggestion
- Eğer onlara biraz sevgi vererek biraz zaman ayırmaya cesaretin varsa, Pencil & Paper harika bir [tablolarla alakalı ingilizce makale](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) yayınladı.  Verilerinizi nasıl biçimlendirmeniz gerektiği konusunda basit ancak büyük farklar yaratabilecek iyi alışkanlıklar öneriyorlar. Bu, bir tabloya karşı bir sonraki sefer sizi rahatsız ediyorsa akılda bulundurulmaya değer fikirlerdir; bunlar bir tablo için mutlaka yapılması gerekenlerin bir listesi değil, ancak düşünmeye değer fikirlerdir.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 7]
```diff
@@ -1,48 +1,46 @@
-### Introduction
+### Giriş
 
-HTML Tables allow you to create two-dimensional tables made of rows and columns. In this lesson, you will learn everything you need to know about HTML Tables.
+HTML tabloları sütunlar ve satırlardan oluşan iki boyutlu yaratmanıza olanak sağlar. Bu derste, HTML tablolarıyla ilgili bilmeniz gereken her şeyi öğreneceksiniz.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md]:
```suggestion
HTML tabloları sütunlar ve satırlardan oluşan iki boyutlu tablolar yaratmanıza olanak sağlar. Bu derste, HTML tablolarıyla ilgili bilmeniz gereken her şeyi öğreneceksiniz.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 8]
```diff
@@ -1,48 +1,46 @@
...
+Bu codepen örneğinde, iki tane satır içeren bir tablo öğesi yarattık. İlk sütun iki başa sahip iken ikinci sütun iki veri hücresine sahip.
 
-### Assignment
+### Ödev
 
 <div class="lesson-content__panel" markdown="1">
 
-1.  Read these [Tables Basics](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics) and [Tables Advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced) tutorials from MDN. They should teach you all the syntax behind HTML tables. It's pretty straightforward. Make sure to code along!
-2.  Do this [Tables Assessment](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Structuring_planet_data) from MDN. It's good to put your newly learned skills to practice!
+1. MDN'den [Tables Basics](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics) ve [Tables Advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced) öğreticilerini okuyunuz. Bu makaleler HTML tablolarının arkasında olan tüm sözdizimini öğretecektir ve fazlasıyla kolaydır. Okuduğunuz makalelerle birlikte kodladığınızdan emin olun!
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md]:
```suggestion
1. MDN'den [Tables Basics](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics) ve [Tables Advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced) adlı ingilizce makaleleri okuyun. Bu makaleler HTML tablolarının arkasında olan tüm sözdizimini öğretecektir ve fazlasıyla kolaydır. Okuduğunuz makalelerle birlikte kodladığınızdan emin olun!
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 9]
```diff
@@ -1,48 +1,46 @@
...
+2. MDN'den [Tables Assessment](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Structuring_planet_data) yapınız. Yeni öğrendiğiniz becerileri pratiğe dökmek güzel!
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md]:
```suggestion
2. MDN'den [Tablo ödevlerini](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Structuring_planet_data) yapınız. Yeni öğrendiğiniz becerileri pratiğe dökmek güzel!
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 10]
```diff
@@ -1,64 +1,62 @@
-### Introduction
+### Giriş
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md:5]:
```suggestion
---
title: 'Emmet'
---

### Giriş
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 11]
```diff
@@ -1,64 +1,62 @@
...
 
-Emmet is a plugin, built into VS Code, that helps you write HTML and CSS more efficiently by providing a bunch of clever shortcuts. By now, you have written a good amount of HTML and CSS, and if you have been using VS Code there's a big chance that you have already encountered Emmet in some way.
+Emmet, VS Code içinde inşaa edilmiş bir eklentidir. HTML ve CSS'i daha efektif yazmaya yarayan Emmet, içinde birçok akıllı kısayollar bulunur. Şimdiye kadar halihazırda birçok HTML ve CSS yazdınız ve VS Code'u kullanıyorsanız Emmet'le bir şekilde karşılaşmış olma ihtimaliniz yüksektir.
 
-### Learning outcomes
+### Öğrenme Çıktıları
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md]:
```suggestion
### Derse genel bakış
```

Bu bölüm original dosyada değiştirilmiş

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 12]
```diff
@@ -1,64 +1,62 @@
...
 
-By the end of this lesson, you should be able to do the following:
+Bu dersin sonunda, bunları biliyor olacaksınız:
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md]:
```suggestion
Bu bölüm, bu derste öğreneceğiniz konuların genel bir özetini içerir.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 13]
```diff
@@ -1,64 +1,62 @@
...
 -   [What syntax would you use to create three elements that have the same class name?](https://docs.emmet.io/cheat-sheet/)
 
-### Additional resources
+### Ek kaynaklar
 
-This section contains helpful links to other content. It isn't required, so consider it supplemental.
+Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
 
--   [Emmet Keybindings](https://marketplace.visualstudio.com/items?itemName=agutierrezr.emmet-keybindings) by [Andrés Gutiérrez](https://agutierrezr.github.io/), is a set of Emmet keybinds for Visual Studio Code. It can be used as a pre-defined group of keybinds, in case you don't know what to map to which keys.
+- [Emmet Keybindings](https://marketplace.visualstudio.com/items?itemName=agutierrezr.emmet-keybindings) tarafından [Andrés Gutiérrez](https://agutierrezr.github.io/), Visual Studio Code için bir Emmet tuş bağlamaları setidir. Hangi tuşa ne atanacağını bilmiyorsanız, bu önceden tanımlanmış bir tuş bağlamaları grubu olarak kullanılabilir.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md]:
```suggestion
- [Emmet Keybindings](https://marketplace.visualstudio.com/items?itemName=agutierrezr.emmet-keybindings), [Andrés Gutiérrez](https://agutierrezr.github.io/) tarafından  hazırlanan Visual Studio Code için bir Emmet tuş bağlamaları setidir. Hangi tuşa ne atanacağını bilmiyorsanız, bu önceden tanımlanmış bir tuş bağlamaları grubu olarak kullanılabilir.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md [hunk 14]
```diff
@@ -1,64 +1,62 @@
...
 
--   [Emmet Live](https://marketplace.visualstudio.com/items?itemName=ysemeniuk.emmet-live) by [Yurii Semeniuk](https://github.com/semeniuk), is another extension in Visual Studio Code. It can continually generate the corresponding HTML structure while you're editing Emmet abbreviations. For the desired outcome, try selecting a random piece of text before calling out this extension.
+- [Emmet Live](https://marketplace.visualstudio.com/items?itemName=ysemeniuk.emmet-live) tarafından [Yurii Semeniuk](https://github.com/semeniuk), Visual Studio Code için başka bir uzantıdır. Emmet kısaltmalarını düzenlerken sürekli olarak karşılık gelen HTML yapısını oluşturabilir. İstenen sonuç için bu uzantıyı çağırmadan önce rastgele bir metin parçası seçmeyi deneyin.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/emmet.md]:
```suggestion
- [Emmet Live](https://marketplace.visualstudio.com/items?itemName=ysemeniuk.emmet-live), [Yurii Semeniuk](https://github.com/semeniuk) tarafından hazırlanan Visual Studio Code için başka bir uzantıdır. Emmet kısaltmalarını düzenlerken sürekli olarak karşılık gelen HTML yapısını oluşturabilir. İstenen sonuç için bu uzantıyı çağırmadan önce rastgele bir metin parçası seçmeyi deneyin.
```

**Review by gulech — APPROVED** [review]
Ellerine sağlık Doğuy, emeklerin için çok teşekkürler. Mükemmel çeviri :)

**Pull request merged**